pkgs/applications/audio/clementine/default.nix

```diff
@@ -50,12 +50,12 @@ let
   withCloud = config.clementine.cloud or true;
 in mkDerivation {
   pname = "clementine";
-  version = "unstable-2022-04-11";
+  version = "1.3.1-unstable-2023-10-02";
 
   src = fetchFromGitHub {
     owner = "clementine-player";
     repo = "Clementine";
-    rev = "250024e117fbe5fae7c62b9c8e655d66412a6ed7";
+    rev = "7b678f26e07faeeea7c3fc61d147c9693e29aa32";
     sha256 = "06fcbs3wig3mh711iypyj49qm5246f7qhvgvv8brqfrd8cqyh6qf";
```

Comment:
Also, this hash needs to change, or you're just building the old rev

Reply:
Sorry, not sure how I missed that :see_no_evil: